content/doc/book/blueocean/activity.adoc

```diff
@@ -23,12 +23,12 @@ The Activity View includes the <<getting-started#navigation-bar, standard naviga
 at the top, with a local navigation bar below that.
 The local navigation bar includes:
 
-* *Pipeline Name* - Clicking on this displaysx to the
-<<#activity, default activity tab>>
+* *Pipeline Name* - Clicking on this displays the
+<<activity, default activity tab>>
```

Comment:
This will now link to the `activity` anchor on this page.  Before it show the literal string "<<#activity, default activity tab>>".